atc/engine/check_delegate.go

```diff
@@ -70,6 +70,13 @@ func (d *checkDelegate) Initializing(logger lager.Logger) {
 	logger.Info("initializing")
 }
 
+func (d *checkDelegate) BeforeSelectWorker(logger lager.Logger) error {
```

Comment:
Why this is needed? It has been implemented in build_step_delegate.

Reply:
Ah it was added by previous commit that for init DB before var source interpolation. It is not needed anymore.